apps/content-publishing-worker/src/batch_announcer/batch.announcer.ts

```diff
@@ -69,6 +69,17 @@ export class BatchAnnouncer {
     return { id: batchId, schemaId, data: { cid, payloadLength: size } };
   }
 
+  public async announceExistingBatch(batch: IBatchFile): Promise<IPublisherJob> {
```

Comment:
nit: I think this is mostly fetching it instead of announcing it so the name is a bit confusing.

Reply:
Just following the existing nomenclature here; the existing `announce` function is _creating_ a batch file & pinning to IPFS, then creating the payload for the on-chain announcement.

This new function skips creating the batch & pinning it, since that's already been done, and simply creates the announcement payload.

I agree that the naming doesn't strictly describe what's being done, but didn't want to get into a whole nomenclature change in this PR.